### ISSUE TITLE
PDF Images

### ISSUE THREAD

**Issue by rossedman** [thujohn/pdf-l4#13]
Hello,

I have followed all the instructions per the wiki and as well as this thread:
https://github.com/thujohn/pdf-l4/issues/12

I have GD enabled, I have published and set the config file and it is incredibly slow. I have to wait about 5 minutes to download or view the PDF. Once this occurs the image still does not work, even though the URL is correct.

I have tried this on multiple machines, all are local developments so servers should not be an issue yet.

Any ideas?
## EDIT

To add to this I noticed that when I turned DOMPDF_ENABLE_REMOTE on it now just runs slow no matter what, even if I am loading nothing! Should I run this in a queue or cache it? I was just trying to style the output.

**Comment by thujohn** [comment]
Hi.

Can I have a link ?

**Comment by rossedman** [comment]
Developing locally so not possible unfortunately

**Comment by thujohn** [comment]
I just created a space to test it : http://lab.packalyst.com/pdf/image
For me it's instant.

**Comment by rossedman** [comment]
Yeah that loaded instantly for me as well. Do you have a code snippet I can inspect? Thanks for your help as well. I really appreciate your time.

**Comment by thujohn** [comment]
You can check this : https://github.com/thujohn/lab
